src/GraphQL/GraphQL.csproj

```diff
@@ -16,8 +16,7 @@
     <PackageReference Include="GraphQL-Parser" Version="5.3.3" />
     <PackageReference Include="Microsoft.CSharp" Version="4.7.0" />
     <PackageReference Include="System.ComponentModel.Annotations" Version="4.7.0" />
-    <PackageReference Include="System.Reactive.Core" Version="4.4.1" />
-    <PackageReference Include="System.Reactive.Linq" Version="4.4.1" />
+    <PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="4.7.1" />
```

Comment:
We should revert the decimal "enhancement" made recently and remove this reference.

Reply:
Why so? I would not. There has already been dependency on `System.Runtime.CompilerServices.Unsafe` from `System.Reactive`, so... we are not losing anything. This is a small package.

Reply:
This has nothing to do with this PR -- go ahead and merge this PR -- we can discuss separately.